526 landing page

cvalarida opened the pull request:
# Scope
This is basically everything we need, but there are still some things we can do to improve upon it including:
- Adding `evss-claims` gating analytics (@djmunoz1 requested this)
- Adding validation

I also have a few content questions for Peggy.

# Screenshots

## Full page
![image](https://user-images.githubusercontent.com/12970166/43433996-b102cbe2-942e-11e8-855c-f70e2b86a8ca.png)

@peggygannon What should the title say?

## Not logged in
![image](https://user-images.githubusercontent.com/12970166/43434207-894f1cb2-942f-11e8-8d4f-ee4d73dccc7f.png)

This is where the rest of the alert boxes will show up.

## No `evss-claims` service
![image](https://user-images.githubusercontent.com/12970166/43434088-1429de9a-942f-11e8-88a7-8676b00a063d.png)

## Submission success
![image](https://user-images.githubusercontent.com/12970166/43434653-808122e0-9431-11e8-8dd2-222637206788.png)

@peggygannon @goldenmeanie What should this box say?

## Submission failure
![image](https://user-images.githubusercontent.com/12970166/43434103-21f99d62-942f-11e8-89f7-873d470d9f4d.png)
@peggygannon What should this box say? (They should be able to try again later.)

Commit:
Hide the form from the sidebar

Commit:
Create landing page

Commit:
Scope the submit function properly

Commit:
Set the bloody content type

peggygannon reviewed (changes requested):
Hi, @cvalarida A couple of tweaks: 

--for the title, I'd like to suggest: 
Increased Disability Compensation Testing

--for the blue button, please made the button language:  
Submit Your Email

--for the EVSS message, can you remove the hyphen between Monday - Friday and add "through". 
It should say "Monday through Friday,"

--Submission message:  
Thank you. We received your email address. We'll contact you to schedule a time to try out the claim for increase tool. 

--Failure message: 
We're sorry. We didn't receive your email address. We're working to fix the problem. Please try again later.

AZSchneider commented:
@peggygannon Can we take "testing" out of the title and the nav? This is a real submission and I don't want people to think it would just be a test and not an actual submission

peggygannon commented:
@AZSchneider Yes, thanks for clarifying that (see my suggestion below). Also, this brings up another question, should this page nest under the Disability "How to Apply" page? 

@cvalarida Here's a new header. 
Apply for Increased Disability Compensation

markgreenburg commented on content/pages/disability-benefits/apply/form-526-disability-claim.md:
```diff
@@ -2,7 +2,8 @@
 layout: page-react.html
-description: Learn how to apply online for increased disability compensation. 
+description: Learn how to apply online for increased disability compensation.
+hideFromSidebar: true
```
Nice!

markgreenburg commented on src/applications/disability-benefits/526EZ/components/create526EmailForm.jsx:
```diff
@@ -0,0 +1,130 @@
+        <AlertBox
+          isVisible
+          headline="To use our new tool, you’ll need to sign in with your premium My HealtheVet or DS Logon account"
+          content="If you don’t have a verified account, you can create an ID.me account to complete the verification process."
```
Should we actually link to idme here?

cvalarida replied:
Ideally? Yeah. I don't know that it's a problem like this because it's a temporary page, though.

markgreenburg commented on src/applications/disability-benefits/526EZ/components/create526EmailForm.jsx:
```diff
@@ -0,0 +1,130 @@
+      );
+    }
+
+    if (!hasEVSSService) {
```
Between this and `isLoggedIn`, we may want to actually use `RequiredLoginView` and then just check for an `isDataAvailable` prop here. But...that's also more complicated, and I don't really see a problem with doing things this way, especially in the short-term.

cvalarida replied:
I thought about it but decided against it for a few reasons:

1) This is super simple
2) `RequiredLoginView` doesn't gate based on the service list provided, so we'd still have to gate it manually
3) I don't know whether the `RequiredLoginView` component is already in the static page bundle, but if it isn't, I didn't want to add it

markgreenburg reviewed (approved):
Arguable whether we're duplicating RequiredLoginView functionality here, but 🤷‍♂️ 
Also, didn't see any tests. We can probably throw those into a separate issue to get this out the door.

markgreenburg commented:
@cvalarida one other enhancement to consider is whether or not we should tell veterans that they've already submitted their email when they reload the page. Currently, since we're not checking anything ahead of time, we always show an empty email input field without confirming that we've previously gotten a successful request from them.

Commit:
Copy edit

Commit:
Add comma

Commit:
Apostrophe

cvalarida commented:
Updated screenshots:

![image](https://user-images.githubusercontent.com/12970166/43474115-d1a51c60-94a6-11e8-92f6-41455fe63bc7.png)
I updated the title and changed the not-logged-in alert to be an info alert instead of an error. I channeled my inner @goldenmeanie and thought that would trigger less of a stress response. (Or something like that, anyhow...

![image](https://user-images.githubusercontent.com/12970166/43474256-33aa0056-94a7-11e8-8da7-03ace016428d.png)
Updated the button content

![image](https://user-images.githubusercontent.com/12970166/43474406-a491b458-94a7-11e8-9566-433141268ec8.png)
Updated success message

![image](https://user-images.githubusercontent.com/12970166/43474449-c6831610-94a7-11e8-8297-53d77c8db75a.png)
Updated failure message

![image](https://user-images.githubusercontent.com/12970166/43474493-eef8aa06-94a7-11e8-9513-6b9b00300fdb.png)
Updated EVSS message

Commit:
Copy edit

cvalarida commented:
![image](https://user-images.githubusercontent.com/12970166/43476085-87e7ddb4-94ac-11e8-8d47-e5377d4f1d5a.png)
